webapp/packages/plugin-connections/src/ConnectionForm/SSH/ConnectionSSHTabService.ts

```diff
@@ -192,8 +192,28 @@ export class ConnectionSSHTabService extends Bootstrap {
         config.networkHandlersConfig = [];
       }
 
+      this.trimSSHInputConfig(handlerConfig);
       config.networkHandlersConfig.push(handlerConfig);
     }
+
+    if (config.networkHandlersConfig?.length) {
+    }
+  }
+
+  private trimSSHInputConfig(input: NetworkHandlerConfigInput) {
```

Comment:
probably we should rename it to the getPreparedSSHConfig and return object from it, because its kind of hard to keep track of object mutations this way
`const preparedSSHConfig = this.getPreparedSSHConfig(handlerConfig);`